src/hexo/themes/sonarwhal/helper/index.js

```diff
@@ -269,8 +269,10 @@ module.exports = function () {
         isNotSectionIndexPage: (page) => {
             return !isSectionIndexPage(page);
         },
-        isPass: (status) => {
-            return status === jobStatus.finished;
+        isPass: (status, overralstatistics) => {
+            const passStatistics = (overralstatistics.errors === 0) && (overralstatistics.warnings === 0);
+
+            return status !== (jobStatus.error) && passStatistics;
```

Comment:
you don't need parenthesis in `jobStatus.error` in case you want to use them (I like to use them for readability), you have to do `(status !== jobStatus.error)`

Reply:
@sarvaje You are right, I meant to add parenthesis for the entire thing ... Thanks!